src/earl-loader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const globby = require('globby')
+
+/**
+ * Load jsonld/earl report(s)
+ * - if there are multiple reports, this function concatanates them into one report
+ *
+ * @method earlLoader
+ * @param {String|String[]} pathOrGlob file path to jsonld/earl report
+ * @returns {Object[]}
+ */
+const earlLoader = async pathOrGlob => {
+	if (!pathOrGlob) {
+		throw new Error(`argument is missing`)
```

Comment:
Suggest you use `assert`, instead of an if / throw. It's functionally the same, but it better expresses what this is doing.